_docs/generator/main.go

```diff
@@ -15,14 +15,32 @@ import (
 
 func main() {
 	_, thisFile, _, _ := runtime.Caller(0)
+
+	// Document the V1 integrations
 	gen := Generator{
-		Dir:          filepath.Join(thisFile, "..", "..", "content", "docs", "dd-trace-go", "integrations"),
+		Dir:          filepath.Join(thisFile, "..", "..", "content", "docs", "dd-trace-go", "v1"),
 		ConfigSource: filepath.Join(thisFile, "..", "..", "..", "instrument"),
+		Validate:     true,
+		CommonPrefix: "gopkg.in/DataDog/dd-trace-go.v1/",
+	}
+	if err := gen.Generate(); err != nil {
+		log.Fatalln(err)
+	}
+
+	// Document the V2 integrations
+	gen = Generator{
+		Dir:          filepath.Join(thisFile, "..", "..", "content", "docs", "dd-trace-go", "v2"),
+		ConfigSource: filepath.Join(thisFile, "..", ".."),
+		Validate:     false, // Currently one aspect is not valid in V2 (rueidis)
```

Comment:
Shall we add an issue for this? So that we can remember to enable the validation when the problem is solved.

Reply:
We migth. The V2 `main` branch is already fixed (targeting v2.1.0`)

Reply:
Created: https://github.com/DataDog/orchestrion/issues/605